Makefile

```diff
@@ -45,6 +45,7 @@ update-kinto-admin:
 	rm -fr kinto_admin/static/*
 	npm install -g kinto-admin
 	kinto-admin build -d kinto_admin/static/
+	sed -i 's/t\.c=n,t\.p=".*"/t\.c=n,t\.p="\/v1\/admin\/"/g' kinto_admin/static/bundle.js
```

Comment:
This seems highly risky, unreliable and not future proof to me... Also this really wants to be updated in the CSS file (`styles.css`) if we really want to fix the issue with missing icons :)